.github/workflows/release_community_cromwell.yml

```diff
@@ -0,0 +1,99 @@
+name: Release Community Cromwell
+
+on:
+  workflow_dispatch: 
+
+jobs:
+  cromwell-release:
+    name: Release Community Cromwell
+    runs-on: ubuntu-latest
+    steps:
+    - uses: sbt/setup-sbt@v1
+
+    # `DSDEJENKINS_PASSWORD` auto syncs from vault with https://github.com/broadinstitute/terraform-ap-deployments/pull/614
```

Comment:
Do we know if this secret will stay put in vault, or if devops has plans to move it to GSM instead?

Reply:
No idea, it may already be coming from GSM - I can look and update the comment. This was copied and pasted from a different GHA we have.

Reply:
Looks like it's still in Vault.

Reply:
Copy-pasted could be a candidate for making a reusable ("composite") action.

Reply:
True, though I don't know that that would result in many fewer lines copied and pasted, this is pretty minimal.